### ISSUE TITLE
Modular builds for Rawhide should go through Bodhi

### ISSUE THREAD

**Issue by ppisar** [fedora-infra/bodhi#4318]
A Fedora user spotted <https://pagure.io/modularity/issue/177> that Rawhide modular builds are not automatically submitted to Bodhi in contrast to classical RPM builds.

**Comment by mattiaverga** [comment]
I've nearly no experience in Fedora Modular or Fedora Flatpaks or Fedora Container and I'm really confused about the current situation.

I tried a couple of times to raise a question on Matrix chat about why after F35 was released the F35 Flatpaks and F35 Containers were not moved to stable as well, but I did not received any reply.
Also, I see no F36 Modular nor F36 Flatpaks exists yet. I'm not sure how F36 Modular builds are currently pushed out.

About automatically creating Rawhide modular updates, I'm not sure the actual automatic_updates consumer can handle update types other than RPMs. It will probably need a deep refactoring.

**Comment by ppisar** [comment]
Modules are always built against a specific Fedora release and they run-require the release. Hence it's not possible to inherit F35 modular builds for F36.

Currently all non-Rawhide modular builds (F<36 now) must be manually submitted to Bodhi. That's expected and fine. Rawhide modular builds go automatically to a compose. This is how it worked with RPM builds up to recently.

But recently there was a change that Rawhide RPM builds are automatically submitted to Bodhi for testing, undergo CI tests and if passed, they are pushed into a Rawhide repository. This change has not been implemented for modular builds.

Thanks for the explanation that the problem is in automatic_updates consumer which does not support modular builds now. Is this issue tracker correct for this request or does the automatic_updates consumer has it's own issue tracker somewhere else?

**Comment by mattiaverga** [comment]
> Modules are always built against a specific Fedora release and they run-require the release. Hence it's not possible to inherit F35 modular builds for F36.
> 
> Currently all non-Rawhide modular builds (F<36 now) must be manually submitted to Bodhi. That's expected and fine. Rawhide modular builds go automatically to a compose. This is how it worked with RPM builds up to recently.
> 
> But recently there was a change that Rawhide RPM builds are automatically submitted to Bodhi for testing, undergo CI tests and if passed, they are pushed into a Rawhide repository. This change has not been implemented for modular builds.
> 

Yep, I was just pointing out that the F36 Modular release doesn't even exist in Bodhi.

> Thanks for the explanation that the problem is in automatic_updates consumer which does not support modular builds now. Is this issue tracker correct for this request or does the automatic_updates consumer has it's own issue tracker somewhere else?

This is the correct place, the automatic_updates consumer lives in Bodhi.
